modules/ROOT/pages/policy-mule3-available-policies.adoc

```diff
@@ -18,8 +18,7 @@ If a policy is already applied, or if the protection it provides is already fulf
 
 == Policy Categories
 
-The following table lists the policy name, the category and the characteristic the policy fulfills. +
-You can only one policy of each characteristic.
+Policies are categorized by the function they perform. For example, because you are able to resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the default policies by its category and the function it performs. 
```

Comment:
```suggestion
Policies are categorized by the function they perform. For example, because you can resolve issues with the help of logs, the Message Logging policy is categorized as a troubleshooting policy. The following table lists the default policies by each policy's category and the function it performs.
```

Reply:
Done! Thanks!